pkg/history/populate_headers.go

```diff
@@ -24,13 +24,15 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func PopulateMissingHeaders(blockChain core.BlockChain, headerRepository datastore.HeaderRepository, startingBlockNumber int64) (int, error) {
-	lastBlock, err := blockChain.LastBlock()
+func PopulateMissingHeaders(blockChain core.BlockChain, headerRepository datastore.HeaderRepository, startingBlockNumber,
+	validationWindow int64) (int, error) {
```

Comment:
Similar nitpicky comment as in headerSync - I wonder if it would make sense to change this argument to `validationWindowSize`  - to me `validationWindow` suggests that it's a range. Also totally cool if we choose not to make this change.

Reply:
Also, looks like this was auto-wrapped - wondering if we want to override that and bring `validationWindow int64) (int, error) {` on the same line

Reply:
I had done the wrapping intentionally because I felt like the line was getting too long 😅 

Happy to remove it